docs/userguide/advanced_data_comparison.md

```diff
@@ -250,8 +250,9 @@ FAILURE
   at "anonymous block", line 15
 ```
 **Note**
-> `join_by` matcher is much faster on performing data comparison, consider using `join_by` over unordered
-> `contain` matcher is not considering order of compared data-sets. Using `unordered` makes no difference (it's default)
+> Consider using `join_by` over unordered `equal`, as the `join_by` matcher is much faster at performing data comparison.
```

Comment:
Consider using `join_by( columns... )` over `unordered()` with `to_equal` matcher. The `join_by` method is much faster at performing data comparison.

Reply:
Corrected. I just changed "with `to_equal` matcher" to "with the `equal` matcher", because in other parts of the docs (e.g. in [`expectations.md`](https://github.com/utPLSQL/utPLSQL/blob/develop/docs/userguide/expectations.md#equal)) the matchers are named without the `to_` prefix.

However, now I wonder whether the part "with the `equal` matcher" is necessary at all.